physical/consul/consul.go

```diff
@@ -150,11 +151,22 @@ func NewConsulBackend(conf map[string]string, logger log.Logger) (physical.Backe
 
 	// Get the additional tags to attach to the registered service name
 	tags := conf["service_tags"]
-
 	if logger.IsDebug() {
 		logger.Debug("physical/consul: config service_tags set", "service_tags", tags)
 	}
 
+	// Get the service-specific address to override the use of the HA redirect address
+	var serviceAddr *string
+	serviceAddrStr, ok := conf["service_address"]
+	if ok {
+		serviceAddr = &serviceAddrStr
+	} else {
```

Comment:
This else block is unnecessary as nil is the default value.

Reply:
@jefferai thanks! Just removed the else block 👍